reference/docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -208,3 +208,15 @@ ALTER TABLE pasystem_banner_dismissed DROP COLUMN user_eid;
 --
 alter table GB_GRADABLE_OBJECT_T alter column IS_EXTRA_CREDIT drop default;
 alter table GB_GRADABLE_OBJECT_T alter column HIDE_IN_ALL_GRADES_TABLE drop default;
+
+-- BEGIN SAM-3066 remove unecessary indexes because Hibernate always create an index on an FK
+ALTER TABLE SAM_PUBLISHEDITEM_T DROP INDEX SAM_PUBITEM_SECTION_I;
+ALTER TABLE SAM_PUBLISHEDITEMFEEDBACK_T DROP INDEX SAM_PUBITEMFB_ITEM_I;
+ALTER TABLE SAM_PUBLISHEDITEMMETADATA_T DROP INDEX SAM_PUBITEMMETA_ITEM_I;
+ALTER TABLE SAM_PUBLISHEDITEMTEXT_T DROP INDEX SAM_PUBITEMTEXT_ITEM_I;
+ALTER TABLE SAM_PUBLISHEDSECTION_T DROP INDEX SAM_PUBSECTION_ASSESSMENT_I;
+ALTER TABLE SAM_PUBSECTION_ASSESSMENT_I DROP INDEX SAM_PUBITEM_SECTION_I;
```

Comment:
SAM_PUBSECTION_ASSESSMENT_I is the index name not the table name. I don't understand what you are reporting here.